cmd/mesh/manifest-versions_test.go

```diff
@@ -53,14 +53,19 @@ func TestGetVersionCompatibleMap(t *testing.T) {
 	if err := yaml.Unmarshal(b, &vs); err != nil {
 		t.Fatal(err)
 	}
-	var curCm, ver133Cm *version.CompatibilityMapping
+	var curCm, ver133Cm, latestCm *version.CompatibilityMapping
 	for i := range vs {
 		if binversion.OperatorBinaryGoVersion.Equal(vs[i].OperatorVersion) {
 			curCm = &vs[i]
 		}
 		if goVer133.Equal(vs[i].OperatorVersion) {
 			ver133Cm = &vs[i]
 		}
+		if latestCm == nil ||
+			(vs[i].OperatorVersion.GreaterThan(latestCm.OperatorVersion) &&
+				vs[i].OperatorVersion.LessThanOrEqual(binversion.OperatorBinaryGoVersion)) {
+			latestCm = &vs[i]
+		}
```

Comment:
It seems like we're copying logic here that should be exercised in the code being tested.

Reply:
fixed.